### DISCUSSION TITLE
Local planning authority

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#36] [Planning consideration]
# Local planning authority

A local planning authority (LPA) is the local government body that is empowered by law to exercise urban planning functions for a particular area ([see Wikipedia](https://en.wikipedia.org/wiki/Local_planning_authority)).

## What we've done

We've created [a schema for Local Authorities](https://raw.githubusercontent.com/digital-land/specification/main/content/dataset/local-authority.md) which defines what data we'd like each local authority record to include. This builds on the [organisation typology](https://raw.githubusercontent.com/digital-land/specification/main/content/typology/organisation.md) by widening out the number of attributes we expect for organisations that are of type `local-authority`.

An associated schema, [local-authority-district](https://raw.githubusercontent.com/digital-land/specification/main/content/dataset/local-authority-district.md), defines how we expect to record the boundaries (or geometries) for each local authority.

## Why

We need to use the Local Planning Authority data ourselves. All data on the [planning.data platform](https://www.planning.data.gov.uk/) has an owner - the organisation responsible for it. A lot of the data is owned and published by Local Planning Authorities so we needed this dataset so that we could assign data to the correct owner and have data about the owner.

We were doing the work to construct this dataset and thought it would be useful to open it up.

## What we're looking for

We'd like to know

* if the data is useful, and how?
* what would improve the data?
* where is the data being used, and how?
* anything else you think we should know about Local Planning Authority data

**Comment by colmjude** [comment]
When we first created this dataset, we collected most of it from a government-managed register. Unfortunately, this has been shut off, making it harder to maintain.

We are still trying to work out how to manage this effectively and sustainably.

**Comment by colmjude** [comment]
## Roles of authorities

We are trying to break done the different roles that different authorities have. For example, we've heard about billing authorities, charging authorities and education authorities. We expect there are more.

We want to document the rules around these roles so that it can be easily captured in the data.

### Education Authority

[missing responsibilities of education authority]

**County authorities** are education authorities
**Unitary authorities** are education authorities

**Combined Authorities** are more complicated. For example, Greater Manchester CA/West of England CA, is the education authority, whereas the responsibility lies with the metropolitan boroughs within West Yorkshire CA

### Charging Authority

A charging authority is an authority that sets CIL. 

It is usually a district or unitary authority.
It is different in London where there is also Mayoral CIL. So the GLA are the charging authority in this case.

### Collecting Authority

The authorities that collect the CIL payments.

All authorities that have their own CIL schedule AND any authorities that collect Mayoral CIL in London

### Mineral and Waste Authority

Mineral and Waste authorities are expected to produce a mineral and waste plan.

**County authorities** are mineral and waste authorities
**Unitary authorities** are mineral and waste authorities

### Highways authorities

[missing responsibilities of highways authority]

[missing rules around who is an highways authority]

---

What other roles are we missing?

**Reply by psd** [reply]
Thanks for kicking off this discussion @colmjude!

We have a [role](https://github.com/digital-land/specification/blob/main/content/role.csv) dataset

Which is used by our [provision-rule](https://github.com/digital-land/specification/blob/main/content/provision-rule.csv) list to generate the [provision](https://github.com/digital-land/specification/blob/main/content/provision.csv) dataset ([datasette version](https://datasette.planning.data.gov.uk/digital-land/provision)), the list of datasets a planning-authority is expected to provide.

This is used in our service to help data providers, performance dashboards for reporting on our work and we plan soon to show against each dataset to help data consumers understand the availability and quality of a dataset

**Reply by psd** [reply]
Roles I think are missing from your list include:

* [Lead Local Flood Authority](https://www.local.gov.uk/topics/severe-weather/flooding/local-flood-risk-management/managing-flood-risk-roles-and) ([LLFA](https://en.wikipedia.org/wiki/Flood_and_Water_Management_Act_2010#Lead_Local_Flood_Authority_(LLFA)))
* **Local Resilience Forum** (we have these listed as a separate organisation, many are LAs)
* A **Competent Authority** for various regulations including designating a Special Area of Conservation

but I'm sure there are many more we've not encountered yet.

I think **Minerals Authority** and **Waste Authority** are two separate roles.

And you didn't call out **Local Planning Authority** ;)

**Reply by psd** [reply]
* [Burial Authority](https://www.legislation.gov.uk/ukpga/1972/70/section/214)

**Reply by colmjude** [reply]
Thanks @psd

Okay, lets confirm if Minerals Authority and Waste Authority are two separate roles. I'll check with our SMEs.

> And you didn't call out Local Planning Authority ;)

Ha, thats true. Is the role "local planning authority" or "planning authority"?

**Reply by colmjude** [reply]
This was shared with us and explains [how your council works](https://www.gov.uk/understand-how-your-council-works).

**Reply by clairemiller2** [reply]
I think the best way to think about roles and responsibilities of planning authorities is based upon assumptions for the most common types of local gov systems:

In a two tier system, the relevant authorities include a district council, and a county council (there might also be parish councils in place). The county and district have different roles and responsibilities for planning:

**County council:**
We refer to the county council as:

- The Highways Authority (including highways, active and sustainable transport)
- The Education Authority (including childcare, primary and secondary schools, 16+ education, and SEND provision)
- The Minerals Authority (the County will prepare Minerals and Waste Plans, which form part of the districts development plan)
- The Waste Authority (the County will prepare Minerals and Waste Plans, which form part of the districts development plan)
- Lead Local Flood Authority (the County is responsible for managing flood risk)
- The county council is also responsible for several other planning considerations, such as adult social care and libraries. 

**District council:**

- Planning policy - incl. the local plan and evidence
- Determining planning applications, and carrying out enforcement
- The district council is also responsible for a number of other planning considerations, such as including sports and leisure, open space and community facilities (e.g. village halls).

Where a unitary authority exists, they are responsible for all county and district council responsibilities.

There are other instances where this is not the case:

- e.g. The 32 London Boroughs are metropolitan borough councils, but are essentially unitary authorities. The Greater London Authority (GLA) provides some services, e.g. fire services, policing, transport and strategic planning (through the spatial development strategy i.e. The London Plan). All Local Plans prepared by London Boroughs must conform with the London Plan.
- Combined Authorities - councils have the opportunity to apply to central government to form a combined authority, with a directly elected mayor. Each deal is different, and therefore it is difficult to apply 'rules' to combined authorities, in terms of their responsibilities. 
- Metropolitan Boroughs - essentially unitary authorities, with some responsibilities often falling to joint bodies e.g. transport, waste, fire services

There are so many responsibilities of all types of planning authorities, and setting out eveery responsibility of every instance of organisation will be challenging, particularly where the 'rules' don't apply.

**Comment by psd** [comment]
Let's find legislative definitions for the roles, such as https://www.legislation.gov.uk/ukpga/2023/55/part/3/chapter/1/enacted#term-relevant-planning-authority

I think that sets out planning authority includes Local Planning Authority, has "Minerals and Waste Authority" as well as **hazardous substances authority**.

**Comment by colmjude** [comment]
## Authorities and their administrative boundaries

This is a summary of what we have on the platform as of 2025-02-18.

| Example | authority type | local-authority-district | administrative boundary on platform |
| --- | --- | --- | --- |
| [Adur district council](https://www.planning.data.gov.uk/entity/26) | NMD | [E07000223](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E07000223) | [x] |
| [Bath and North East Somerset Council](https://www.planning.data.gov.uk/entity/38) | UA | [E06000022](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E06000022) | [x] |
| [Liverpool City Council](https://www.planning.data.gov.uk/entity/202) | MD | [E08000012](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E08000012) | [x] |
| [Cumbria County Council](https://www.planning.data.gov.uk/entity/89) | CTY | E10000006 | [] |
| [West Yorkshire Combined Authority](https://www.planning.data.gov.uk/entity/391) | COMB | E47000003 | [] |
| [City of London Corporation](https://www.planning.data.gov.uk/entity/203) | CC | [E09000001](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E09000001) | [x] |
| [London Borough of Bexley](https://www.planning.data.gov.uk/entity/43) | LBO | [E09000004](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E09000004) | [x] |
| [Greater London Authority](https://www.planning.data.gov.uk/entity/144) | SRA | [E12000007](https://www.planning.data.gov.uk/prefix/statistical-geography/reference/E12000007) | [x] |
